src/ucp/wireup/wireup.c

```diff
@@ -190,8 +191,15 @@ static void ucp_wireup_process_request(ucp_worker_h worker, const ucp_wireup_msg
         }
     }
 
+    /* Use default parameters to initialize local lanes.
+     * This might not always work - need to pass some parameters in the wireup
```

Comment:
pls add to TODO / or to this PR:
pass error handling mode in wireup message, in case ep is created internally, as a result of peer request, before the user creates it himself. in this case ep will have err mode "NONE" even though user could have requested err mode "PEER"